packages/nextflow-language-server/package.yaml

```
@@ -0,0 +1,23 @@
+---
+name: nextflow-language-server
+description: A language server for Nextflow.
+homepage: https://github.com/nextflow-io/language-server
+licenses:
+  - MIT
```

```suggestion
  - Apache-2.0
```

This is the correct license